q2cli/tests/test_tools.py

```diff
@@ -200,8 +203,70 @@ def test_export_path_file_is_replaced(self):
         ])
         with open(output_path, 'r') as f:
             file = f.read()
+
         self.assertNotIn('HelloWorld', file)
 
+    def test_export_to_file_with_format_success_message(self):
+        output_path = os.path.join(self.tempdir, 'output.int')
+        result = self.runner.invoke(tools, [
+            'export', '--input-path', self.ints1, '--output-path', output_path,
+            '--output-format', 'IntSequenceFormatV2'
+            ])
+        success = 'Exported %s as IntSequenceFormatV2 to file %s\n' % (
+                   self.ints1, output_path)
+        self.assertEqual(success, result.output)
+
+    def test_export_to_dir_without_format_success_message(self):
+        output_path = os.path.join(self.tempdir, 'output')
+        result = self.runner.invoke(tools, [
+            'export', '--input-path', self.ints1, '--output-path', output_path
+            ])
+        success = 'Exported %s as IntSequenceDirectoryFormat to '\
+                  'directory %s\n' % (self.ints1, output_path)
+        self.assertEqual(success, result.output)
+
+    def test_export_visualization_to_dir_success_message(self):
+        output_path = os.path.join(self.tempdir, 'output')
+        result = self.runner.invoke(tools, [
+            'export', '--input-path', self.viz, '--output-path', output_path
+        ])
+        success = 'Exported %s as Visualization to '\
+                  'directory %s\n' % (self.viz, output_path)
+
+        self.assertEqual(success, result.output)
+
+    def test_extract_to_dir_success_message(self):
+        result = self.runner.invoke(tools, [
+            'extract', self.ints1, '--output-dir', self.tempdir
+            ])
+        success = 'Extracted %s to directory %s' % (self.ints1, self.tempdir)
+        self.assertIn(success, result.output)
+
+    def test_import_from_file_success_message(self):
```

Comment:
Should this test title be `test_import_from_directory_without_format_success_message`?